crowdsec-docs/unversioned/cti_api/taxonomy/false_positives.mdx

```diff
@@ -22,11 +22,21 @@ export const columns = [
 
 <GithubIconRender url={fpURL}></GithubIconRender>
 
+IPs in this category are considered completely safe and trusted. Alerts triggered by these IPs are likely due to misconfiguration or overly sensitive alerting rules.
+
+IPs belonging to those categories will have the `safe` [reputation](/u/cti_api/taxonomy/cti_object#reputation).
+
+:::warning
+
+You might want to investigate any alerts associated with these IPs to ensure your configuration is correct.
```

Comment:
Do we have doubts about FP ?
Ideally we should NOt have doubt if we qualify them as FP with a strict list of genuine crawlers and safe bots